conf/testfm.sample.yaml

```diff
@@ -0,0 +1,17 @@
+# copy this file as `testfm.local.yaml` and override any config provided under `conf/*.yaml`
+# or preferably use environment variables instead of this file
+# example:
+# `export TESTFM_SERVER__SERVER_HOSTNAME=myserver.example.com`
+---
+# SUBSCRIPTION:
```

Comment:
This is very generic name should not be overridden tomorrow if this file makes to `/openshift/conf` in satelliteqe-jenkins tomorrow. If the plan to make it generic for robottelo as well then I am good.

Else rename it to make it specific to `TESTFM`.

Reply:
@jyejare Well, so will be having a subscription.yaml in `openshift/conf` directory, and will call this subscription values from that file using `@format {{this.subscription.*}}"` tokens, so I guess won't conflict there

Reply:
The yaml file name doesnt matter at all the root name in yaml (here `SUBSCRIPTION` ) matters the most ! And I am trying to ensure that it should not repeat. Thats how dynaconf traverse through YAML not with file name.

So rename the root name here or rename file to subscription.yaml where others can contribute for subscriptions :)

Reply:
@jyejare  Kept this root name as `SUBSCRIPTION` in YAML intensionally to get values from `subscription.yaml` which will be available in CI with `testfm.yaml`, these both files will have the same root as a subscription, and all values for subscription in testfm.yaml will be load from subscription.yaml, so whichever gets called, will return the same values for us :)

Reply:
@jyejare Added comment above the section to copy these details from `subscription.yaml` or to include that YAML under `conf/` and remove this section.
I would not prefer to add more dynaconf samples/templates here for testfm, I guess that won't be a blocker for this PR.

Reply:
The definition of multiple top level keys in dynaconf files will result in them being merged, so its fine to have both subscription and testfm maps in this file. The dynaconf configuration includes preloading conf directory, so if separate files are defined, they will be loaded correctly.

This brings testfm's configuration in line with robottelo, and will facilitate consolidating the repo.